code/modules/food_and_drinks/food/foods/sandwiches.dm

```diff
@@ -3,7 +3,22 @@
 //		Burgers		//
 //////////////////////
 
-/obj/item/reagent_containers/food/snacks/brainburger
+/obj/item/reagent_containers/food/snacks/burger // Abstract object used for inheritance. Should never spawn. Needed to not break recipes.
```

Comment:
I was initially going to do that, however, it broke any recipe that used/required just a plain burger and allowed you to stick any burger into it(which for right now is just the big bite burger though it could end up changing). Hence the line about not breaking recipes. I'd be all for changing it if you have suggestions on how to stop that from occurring. I could make it contain/use maybe the cheeseburger instead, I guess, but I preferred to keep recipes the same for the most part.